test/actions/test_image_preprocess_agent_plugin.ts

```
@@ -0,0 +1,60 @@
+// this test is generated by claude code.
+import test from "node:test";
+import assert from "node:assert";
+import type { MulmoStudioContext, MulmoBeat } from "../../src/types/index.js";
```

ESLint

[error] Remove this unused import of 'MulmoStudioContext'.

(sonarjs/unused-import)